### PR TITLE
Add a note about CLI not terminating user's session immediately

### PR DESCRIPTION
**Pull request by jasonblais**
See relevant threads for additional context
 - https://community.mattermost.com/core/pl/b8szcxij8bbr5mak91stnhhaqw
 - https://community.mattermost.com/private-core/pl/ziktozputpg1jpwe5jdxtsbf6w

and GitHub issue which sparked the discussion and this doc PR https://github.com/mattermost/mattermost-server/pull/8699#issuecomment-568611594

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -1749,6 +1749,9 @@ mattermost user deactivate
       ./mattermost user deactivate user@example.com
       ./mattermost user deactivate username
 
+  .. note::
+    Users deactivated via this CLI command can continue to use Mattermost until the user cache is manually refreshed (they are prevented from logging in to Mattermost, however). If you want to immediately terminate the user's session, purge all caches in **System Console > Web Server > Purge All Caches** after running this command, or `use the API <https://api.mattermost.com/#tag/users%2Fpaths%2F~1users~1%7Buser_id%7D%2Fdelete>`_ to deactivate a user account which immediately terminates the session.
```
Comment (wiersgallak) [inline source/administration/command-line-tools.rst]: Some proposed rewording to make a bit more readable: 

Users deactivated via this CLI command can continue to use Mattermost, if they are already logged in, until the user cache is manually refreshed.  Users who are deactivated and not currently logged in are prevented from logging in to Mattermost. If you want to immediately terminate a deactivated user's session, purge all caches in **System Console > Web Server > Purge All Caches** after running this command, or `use the API <https://api.mattermost.com/#tag/users%2Fpaths%2F~1users~1%7Buser_id%7D%2Fdelete>`_ to deactivate a user account which immediately terminates the session.

Comment (justinegeffen) [reply source/administration/command-line-tools.rst]: I've made a few tweaks based on your feedback, @wiersgallak. Not sure if my second sentence clarifies or muddies though. 

Users deactivated via this CLI command can continue to use Mattermost, if they are already logged in, until the user cache is manually refreshed. Users who are deactivated when they're not logged in will not be able to log in to Mattermost again. 

If you want to immediately terminate a deactivated user's session, purge all caches in **System Console > Web Server > Purge All Caches** after running this command. 

You can also use the <https://api.mattermost.com/#tag/users%2Fpaths%2F~1users~1%7Buser_id%7D%2Fdelete>_  command to deactivate a user account and immediately terminate the session.

Comment (wiersgallak) [reply source/administration/command-line-tools.rst]: @justinegeffen This sounds good to me.

Comment (jasonblais) [reply source/administration/command-line-tools.rst]: Thanks all! Updates made per above

File: source/administration/command-line-tools.rst [hunk 2]
```diff
@@ -1749,6 +1749,13 @@ mattermost user deactivate
       ./mattermost user deactivate user@example.com
       ./mattermost user deactivate username
 
+  .. note::
+    Users deactivated via this CLI command can continue to use Mattermost, if they are already logged in, until the user cache is manually refreshed. Users who are deactivated when they're not logged in will not be able to log in to Mattermost again.
```
Comment (DSchalla) [inline source/administration/command-line-tools.rst]:
```suggestion
    Users deactivated via this CLI command can continue to use Mattermost, if they are already logged in, until the user cache is manually purged or automatically after 30 minutes. Users who are deactivated when they're not logged in will not be able to log in to Mattermost again.
```

**Pull request merged**